src/framework/theme/components/actions/actions.component.ts

```diff
@@ -24,6 +24,7 @@ import { convertToBoolProperty } from '../helpers';
     <ng-template #showContent>
       <ng-content></ng-content>
     </ng-template>
+    <nb-badge *ngIf="badgeText" text={{badgeText}} status={{badgeStatus}} position={{badgePosition}}></nb-badge>
```

Comment:
Let's add spaces inside of `{{ }}` statement. Moreover, in this case `[text]="badgeText" [status]="badgeStatus"` should look even better.